src/rpcclient.cpp

```diff
@@ -129,6 +129,9 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "omni_listblocktransactions", 0 },
     { "omni_getorderbook", 0 },
     { "omni_getorderbook", 1 },
+    { "omni_getseedblocks", 0 },
+    { "omni_getseedblocks", 1 },
+    { "omni_getseedblocks", 2 },
```

Comment:
Sorry, think I misread your comment - thought you meant all three weren't required (which I wanted to test because I didn't think it would work).  

Now rereading I realize you meant just the last line hehe - all sorted.